sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/JwtSigned.cs

```diff
@@ -0,0 +1,243 @@
+﻿using System;
+using System.Collections.Generic;
+using JWT;
+using JWT.Algorithms;
+using JWT.Builder;
+
+namespace Tests.Diagnostics
+{
+    class Program
+    {
+        const string secret = "GQDstcKsx0NHjPOuXOYg5MbeJ1XT0uFiwDVvVBrk";
```

Comment:
Would be nice to add a few more tests in:
- constructor
- nested function
- inside lambda
- inside property
- inside interface with default members
...

Reply:
I've added FP and FN for local function. Placing code into different wrapping syntax does not bring value, before rule does not depend on parent wrapper and we don't usually do that in that case.